src/components/cursor.js

```diff
@@ -33,60 +38,118 @@ module.exports.Component = registerComponent('cursor', {
   dependencies: ['raycaster'],
 
   schema: {
+    downEvents: {default: []},
```

Comment:
I'm not 100% sure about this since it sets a precedent of a pattern that won't be available for all the events. If we do this for `downEvents` and `upEvents` why not also for `enter` \ `leave` and why not for all the other components that emit events. this is basically a declarative way to do 
```javascript
el.addEventListener('mousedown', function () { .... emit('xxx') }
```